### PR TITLE
Fix/typo in example

### PR DESCRIPTION
**Pull request by gilacost**
## BEFORE: 
Captura de pantalla 2020-04-12 a las 9 11 17

## AFTER:
Captura de pantalla 2020-04-12 a las 9 11 26

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# HELP
```
Comment (alexdesousa) [inline Makefile]: I wouldn't add this file, considering is not cross-platform and doesn't serve a bigger purpose other than running some standard commands to generate docs. Additionally, it's not related with the PR goal ;)

**Review by alexdesousa — CHANGES_REQUESTED** [review]
Thanks for the fix :)

File: Makefile (hunk 1 above)
Comment (gilacost) [reply Makefile]: make sense

**Comment by gilacost** [comment]
😳

**Pull request merged**